src/lib.rs

```diff
@@ -65,10 +64,14 @@ pub struct RiffFile {
 
 /// Resource Interchange File Format
 impl RiffFile {
-    /// Open a RIFF file
+    /// Open a RIFF file from a filename
     pub fn open(filename: &str) -> Result<Self> {
```

Comment:
The file name parameter type could also be changed from `&str` to `&Path` or `P: AsRef<Path>` here.